### PR TITLE
 AffineCipher.py

### PR DESCRIPTION
**Pull request by riya-joshi-401**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

Affine substitution cipher algorithm implemented in python . 
Contains an encryption and a decryption functions .
Takes a plain text and two keys ( namely k1 , k2 ) as inputs.
Shows the encrypted text  , decrypts it and again shows the decrypted text.( if all the conditions are satisfied)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

### CODE REVIEW

**Comment by riya-joshi-401** [comment]
added affine substitution cipher kindly review it
@SKAUL05 @plazzy99 @Abhijit2505 @Kajol-Kumari @HarshCasper @routrohan

**Review by SKAUL05 — CHANGES_REQUESTED** [review]
Use meaningful variable names instead of alphabets.
Include Sample I/O.

File: Python/cryptography/AffineCipher.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+  key1 ,key2=input("Enter the keys : ").split()
+
+  li=[] # a list for storing the encrypted characters which we later merged to a string i.e cipher_text
+
+
+
+  def gcd(a, b): # for checking coprime
+      while b != 0:
+          a, b = b, a % b
```
Comment (SKAUL05) [inline Python/cryptography/AffineCipher.py]: Change variable names from alphabets to meaningful names in whole script

**Review by SKAUL05 — CHANGES_REQUESTED** [review]

**Review by SKAUL05 — APPROVED** [review]

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]
update readme file for this program

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]
you don't have to create the readme file 
you have to edit the python readme file

**Comment by riya-joshi-401** [comment]
> you don't have to create the readme file
> you have to edit the python readme file

im not getting exactly what is meant by " python readme file " ..please help me with that!!

**Comment by vatsalkesarwani12** [comment]
> > you don't have to create the readme file
> > you have to edit the python readme file
> 
> im not getting exactly what is meant by " python readme file " ..please help me with that!!

check pull request made in the repository by other participant

File: Python/cryptography/AffineCipher.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  plain_text=input("Enter the plain text : ")
+
+  global key1  # key 1
+  global key2  # key 2
+  global cipher_text # to store to the cipher text or the encrypted plain_text
+  cipher_text=''
+  
+  
+
+
```
Comment (Abhijit2505) [inline Python/cryptography/AffineCipher.py]: Please remove spaced from line 10-13 of your code. They are not needed.

File: Python/cryptography/AffineCipher.py [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+  key1 ,key2=input("Enter the keys : ").split()
+
+  list_encrypted_characters=[] # a list for storing the encrypted characters which we later merged to a string i.e cipher_text
+
+
+
```
Comment (Abhijit2505) [inline Python/cryptography/AffineCipher.py]: Remove these spaces as well.

**Review by Abhijit2505 — CHANGES_REQUESTED** [review]
Please add your work to the README file as well.

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]
code LGTM but you have to update your readme file.

**Review by vatsalkesarwani12 — APPROVED** [review]

**Review by Abhijit2505 — APPROVED** [review]
It's looking good to me as well.
Kindly Merge this PR

**Comment by iamrajiv** [comment]
@all-contributors please add @riya-joshi-401 for code

**Pull request merged**